### DISCUSSION TITLE
Requesting overflow-wrap on Box

### DISCUSSION THREAD

**Discussion by victoriahu** [twilio-labs/paste#1396] [Ideas] [answered]
We are trying to wrap text with Box. 

<img width="818" alt="Screen Shot 2021-04-28 at 11 08 14 AM" src="https://user-images.githubusercontent.com/15176943/116451988-0d3e5a80-a812-11eb-87cd-848e56b1f4f7.png">

We would like to see overflow-wrap as a property of Box, which is currently not supported. Currently, word-break is supported using the wordBreak attribute on a paste Box.
Word-wrap and word-break are older properties that were replaced by overflow-wrap as far as we can tell. Not sure why Paste supports those instead: https://stackoverflow.com/questions/17259916/difference-between-overflow-wrap-and-word-break

Instead, we would like to have overflow-wrap supported on Box.

**Comment by SiTaggart** [comment]
👋🏼  Thanks for the feature request.

Should be pretty quick to get this added in. As @nkrantz is the DSYS helper this week, this would be an EXCELLENT opportunity to work on `Box` and adding a style prop as a helper task for the next week. Happy to pair with you on it @nkrantz

**Reply by nkrantz** [reply]
Sounds good! @victoriahu - I'll be back with an update within the next few days.

**Comment by TheSisb** [comment, marked as answer]
Just merged this in (https://github.com/twilio-labs/paste/pull/1424), but it isn't published yet. It's coming soon and I'll report back when it's published.

**Reply by TheSisb** [reply]
It's been published :) Update your version of `@twilio-paste/core` for the fix.
